client-java/src/unit-tests/resources/resources/test-data-0.txt

```diff
@@ -0,0 +1,4 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna 
```

Comment:
a) having just `test-data-0` and `test-data-1` is plenty
b) this file is multiline whereas the others are a single line - i presume this is intentional
c) 10 bytes or so here would be plenty, unless you want to proof-of-concept a record larger than 64K (tcp packet max size, has caused problems in the past)

Reply:
I did have it with two files and the tests passed. At some point the tests only failed with more downloads. Admittedly that was before the streaming stuff in the client.
I have cut the number of file down to two, but kept the content the same. One all on one lin and the other with line breaks. Both are ~448 bytes. Test data updated in line with the change.